release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -373,9 +373,9 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 [id="ocp-4-10-cluster-cloud-controller-manager-operator"]
 ==== Cloud controller managers for additional cloud providers
 
-The Kubernetes community plans to deprecate the Kubernetes controller manager in favor of using cloud controller managers to interact with underlying cloud platforms. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms. The Alibaba Cloud and IBM Cloud implementations that are added in this release of {product-title} use cloud controller managers.
+The Kubernetes community plans to deprecate the Kubernetes controller manager in favor of using cloud controller managers to interact with underlying cloud platforms. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms. The IBM Cloud implementation that is added in this release of {product-title} uses cloud controller managers.
 
-In addition, this release supports using cloud controller managers for Google Cloud Platform (GCP) and VMware vSphere as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+In addition, this release supports using cloud controller managers for Google Cloud Platform (GCP), VMware vSphere, and Alibaba Cloud as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
```

Comment:
Perhaps this is my ignorance. This wording _this release supports using cloud controller managers for_ makes it sound like the CCM is an optional feature I can use. Is that right?